### DISCUSSION TITLE
podman-compose how to support other host's mount device

### DISCUSSION THREAD

**Discussion by LinglingXiang** [containers/podman#20301] [Q&A]
i tried to change the docker-compose to podman-compose

before the yaml file has the shared volumes like this 
volumes:
    test-data-nfs:
        driver: local
        driver_opts:
          type: nfs
          o: addr=10.0.148.162,ro,nolock,nosuid,nodev
          device: :/test_data

the two services use it
    tty: true
    stdin_open: true
    security_opt:
      - label:disable 
    # privileged is required by running podman in container
    privileged: true
    volumes:
      - .:/code
      - brewroot-prod-nfs:/mnt/redhat:ro,nocopy
      - containers:/var/lib/containers:Z
      - test-data-nfs:/mnt/test_data:ro,nocopy
    working_dir: /code
    command: /bin/bash

but when start it, it will display 
Error: unable to start container "713a907311297cf0b801bf11d583521f164777743a297d626f86b1e25bd49c3d": mounting volume pub-test_test-data-nfs for container 713a907311297cf0b801bf11d583521f164777743a297d626f86b1e25bd49c3d: mount.nfs: Failed to resolve server : Name or service not known

**Comment by vrothberg** [comment]
Please reach out to https://github.com/containers/podman-compose with podman-compose specific questions.
